tests/testlistselector.py

```diff
@@ -18,7 +18,7 @@ def setUp(self):
         super().setUp()
         class P(param.Parameterized):
             e = param.ListSelector(default=[5],objects=[5,6,7])
-            f = param.ListSelector(default=10)
+            f = param.ListSelector(default=[10])
```

Comment:
As best I can tell this was a typo in the test, previously not caught?

Reply:
Definitely looks like it.